### DISCUSSION TITLE
Receiving sflow data but nothing on UI

### DISCUSSION THREAD

**Discussion by aqib134** [akvorado/akvorado#1350] [Q&A]
Hello, I'm using dockers on ubuntu to run akvorado, I can see UI but no flows can be seen on akvorado UI that I'm receving from a cisco router on port 2055 udp, here's the tcpdump output :

15:09:38.151351 IP router-WIP.60604 > 172.18.0.6.iop: UDP, length 1380 15:09:38.151353 IP router-WIP.60604 > 172.18.0.6.iop: UDP, length 1380
15:09:38.151505 IP router-WIP.60604 > akvorado-server.iop: UDP, length 1340
15:09:38.151512 IP router-WIP.60604 > 172.18.0.6.iop: UDP, length 1340
15:09:38.151515 IP router-WIP.60604> 172.18.0.6.iop: UDP, length 1340
15:09:38.383918 IP router-WIP.68684 > akvorado-server.iop: UDP, length 1236 

here is the error rate I see in inlet service and grep _error:

#HELP akvorado_inlet_core_flows_errors total Number of flows with errors. #TYPE akvorado_inlet_core_flows_errors_total counter
akvorado_inlet_core_flows_errors_total(error="SNMP cache miss", exporter="10.20.0.8") 75880
akvorado_inlet_core_flows_errors_total(error="input and output interfaces missing", exporter="10.20.0.8") 56525 akvorado_inlet_core_flows_errors_total(error="sampling rate missing", exporter="10.20.0.8") 56525 #HELP akvorado_inlet_metadata_provider_snmp_poller_error_requests total Number of failed requests. #TYPE akvorado_inlet_metadata_provider_snmp_poller_error_requests total counter
akvorado_inlet_metadata_provider_snmp_poller_error_requests_total{error="get", exporter="10.20.0.8") 1226 

o/common/httpserver", "message": "HTTP request"}
{"level":"error", "error":"request timeout (after 1 retries)", "exporter":"10.20.0.8","time":"2824-88-12T15:12:16Z","caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp", "message":"unable to GET (4 OIDs)"} {"level":"error", "error":"request timeout (after 1 retries)", "exporter":"10.20.0.8", "time":"2824-88-12T15:12:16Z","caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp", "message":"unable to GET (7 OIDs)"} {"level":"error", "error":"request timeout (after 1 retries)", "exporter":"10.20.0.8","time":"2824-88-12T15:12:16Z","caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp", "message":"unable to GET (4 OIDs)"} {"level":"warn", "exporter":"10.20.0.8","time":"2024-08-12T15:12:23Z","caller":"akvorado/inlet/metadata/root.go:320","module":"akvorado/inlet/metadata", "message": "provider breaker open"} {"level":"info","handler":"/api/","method":"GET", "url":"/api/ve/healthcheck", "ip":"[::1]:46676", "user-agent":"Go-http-client/1.1", "status":200,"size":230, "duration":8.892171, "time":"2824-88-12T15:12:37Z","caller" : "akvorado/common/httpserver/root.go:88", "module":"ak ("level": "info", "handler": "/api/ve/inlet/metrics", "method": "GET", "url":"/api/v8/inlet/metrics","ip":"172.18.0.7:45716", "user-agent":"curl/7.68.8","status":200,"size":8284, "duration":2.558989,"time":"2024-08-12T15:12:54Z", "caller*:*akvorado/common/httpserver/root.go "module":"akvorado/common/httpserver", "message": "HTTP request"} {"level":"info","handler":"/api/", "method":"GET", "url":"/api/ve/healthcheck","ip":"[::1]:40902", "user-agent":"Go-http-client/1.1", "status": 200, "size":230, "duration":0.125192, "time":"2824-88-12T15:13:87Z","caller":"akvorado/common/httpserver/root.go:88", "module":"ak o/common/httpserver", "message": "HTTP request"} ("level": "error", "error":"request timeout (after 1 retries)", "exporter":"10.20.0.8","time":"2824-88-12T15:13:25Z", "caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp", "message":"unable to GET (7 OIDs)"} {"level":"error", "error": "request timeout (after 1 retries)", "exporter": "18.20.0.8", "time":"2824-88-12T15:13:25Z","caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115", "module":"akvorado/inlet/metadata/provider/snmp", "message": "unable to GET (7 OIDs)"} {"level":"error", "error":"request timeout (after 1 retries)", "exporter":"10.20.0.8","time":"2824-88-12T15:13:25Z","caller" : "akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp", "message":"unable to GET (4 OIDs)"}

Can someone help me with that?

**Comment by broonu** [comment]
Hello,

```
akvorado_inlet_core_flows_errors_total(error="SNMP cache miss", exporter="10.20.0.8") 75880
akvorado_inlet_core_flows_errors_total(error="input and output interfaces missing", exporter="10.20.0.8") 56525
akvorado_inlet_core_flows_errors_total(error="sampling rate missing", exporter="10.20.0.8") 56525 #HELP
```

You need to configure SNMP and sampling rate for this device.

**Reply by aqib134** [reply]
Do we need to configure this on router or akvorado server?
